api/AudioTrackList.json

```diff
@@ -77,7 +77,7 @@
             "version_added": "7"
           },
           "samsunginternet_android": {
-            "version_added": null
+            "version_added": "5.0"
```

Comment:
This was enabled behind a flag in Chrome Android. Was this enabled by default in Samsung Internet?

Reply:
I don't think so, i'll mark it false interestingly 
```
"webview_android": {
    "version_added": "45"
}
```